pkg/vm/process/process.go

```diff
@@ -86,6 +86,14 @@ func NewFromProc(p *Process, ctx context.Context, regNumber int) *Process {
 	return proc
 }
 
+// NewFromNil return Process, which behave like (*Process)(nil), see Process.GetMPool
+func NewFromNil(ctx context.Context) *Process {
```

Comment:
I think this is wrong because it will cause mpool to throw oom after a long run.

Reply:
this function is aim to passing `(*process.Process)(nil)` to `colexec.EvalExpr(...)`
in here: https://github.com/matrixorigin/matrixone/pull/7181/files#diff-6808a91f32ea7065cd63ff5887ac4600b5b6744b6cd2bb12859a0e4a5fb0efd2R301
- behaviour same as (*process.Process)(nil)

Reply:
@ouyuanning please help me to fix

Reply:
done, use the real `process.Process` from `table.Ranges(...)`